book/makefile

```diff
@@ -109,16 +110,16 @@ book_serif/book.pdf: $(src_all)
 book_sans_serif/book_sans_serif.pdf: $(src_all)
 	$(LATEX) -output-directory=book_sans_serif book_sans_serif.tex
 
-.PHONY: copy_ebook_files copy_ebook_files_cl
+.PHONY: copy_ebook_files copy_ebook_files_low_res
```

Comment:
That does not look like main? not upstream main at least.

Reply:
As far as I can tell, I am in sync and this was something that just slipped through the cracks when I renamed the _cl options to _low_res [book makefile](https://github.com/hendricius/the-sourdough-framework/blob/main/book/makefile) line 112 & 120